src/metabase/query_processor/middleware/optimize_temporal_filters.clj

```diff
@@ -99,13 +154,16 @@
       (_ :guard (partial mbql.u/is-clause? (set (keys (methods optimize-filter)))))
       (if (can-optimize-filter? &match)
         (let [optimized (optimize-filter &match)]
+          (when-not optimized
```

Comment:
should this perhaps log `&match` and then fall back to the unoptimized filter?

Reply:
yeah that's probably a better situation. I'll change it